contracts/modules/SuperMinterV1_1.sol

```diff
@@ -550,25 +596,6 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
         emit MerkleRootSet(edition, tier, scheduleNum, merkleRoot);
     }
 
-    /**
-     * @inheritdoc ISuperMinterV1_1
-     */
-    function setSigner(
```

Comment:
This seems to be a change where the only signer used going forward is going to be the platform signer for the platform declared as part of the mint creation. This means no more declaring a specific signer, instead for VERIFY_SIGNATURE mint you'll declare a platform and the platform will will use their enrolled signer. Is this correct? The change on line 1119 seems to support this being the case.

Reply:
correct